File: online judge.kaner murillo/python/11734.py
#https://onlinejudge.org/external/117/11734.pdf
#11734

from sys import stdin

casos = int(stdin.readline())

for i in range(1, casos+1):

    cadenaEstudiantes = stdin.readline().rstrip('\n')
    cadenaJuez = stdin.readline().rstrip('\n')

    aux1 = cadenaEstudiantes.split()

    if cadenaEstudiantes == cadenaJuez:

        print(f"Case {i}: Yes")
    
    elif "".join(aux1) != cadenaJuez:

        print(f"Case {i}: Wrong Answer")

    else:

        print(f"Case {i}: Output Format Error")
    

        
        














        
    

        

        

File: online judge.kaner murillo/c++/black list/11734.cpp
#include<cstdio>
#include<iostream>
#include<string>

using namespace std;

int main(){

    int casos, i;  
    string cadena1, cadena2;
    char letra1, letra2;

    scanf("%i", &casos);
    cin.ignore();

    for(i = 1; i<casos+1; i++){
        
        getline(cin,cadena1);
        getline(cin,cadena2);
        
        if(cadena1.length() == cadena2.length() && cadena1 == cadena2){

            printf("Case %i: Yes\n", i);
        }

        else if(cadena1.length() == cadena2.length() && cadena1 != cadena2){

            printf("Case %i: Wrong Answer\n",i);
        }
        else{

            printf("Case %i: Output Format Error\n", i);
        }
    }
    return 0;
}